front/src/reducers/osrdconf2/common/index.ts

```diff
@@ -0,0 +1,350 @@
+    updateRollingStockComfort(state: Draft<S>, action: PayloadAction<S['rollingStockComfort']>) {
+      state.rollingStockComfort = action.payload;
+    },
+    updateSpeedLimitByTag(state: Draft<S>, action: PayloadAction<S['speedLimitByTag']>) {
+      state.speedLimitByTag = action.payload;
+    },
+    updateOrigin(state: Draft<S>, action: PayloadAction<S['origin']>) {
+      state.origin = action.payload;
+    },
+    updateInitialSpeed(state: Draft<S>, action: PayloadAction<S['initialSpeed']>) {
+      state.initialSpeed = action.payload;
+    },
+    updateDepartureTime(state: Draft<S>, action: PayloadAction<S['departureTime']>) {
+      state.departureTime = action.payload;
+    },
+    updateOriginTime(state: Draft<S>, action: PayloadAction<S['originTime']>) {
+      if (action.payload) {
+        const newOriginTimeSeconds = time2sec(action.payload);
+        const { originLinkedBounds, originTime, originUpperBoundTime } = state;
+        if (originLinkedBounds) {
+          const difference =
+            originTime && originUpperBoundTime
+              ? time2sec(originUpperBoundTime) - time2sec(originTime)
+              : ORIGIN_TIME_BOUND_DEFAULT_DIFFERENCE;
+          state.originUpperBoundTime = sec2time(newOriginTimeSeconds + difference);
+        }
+        if (
+          state.originUpperBoundTime &&
+          time2sec(action.payload) > time2sec(state.originUpperBoundTime)
+        ) {
+          state.originTime = state.originUpperBoundTime;
+        } else {
+          state.originTime = action.payload;
+        }
+      }
+    },
+    updateOriginUpperBoundTime(state: Draft<S>, action: PayloadAction<S['originUpperBoundTime']>) {
+      if (action.payload) {
+        const newOriginUpperBoundTimeSeconds = time2sec(action.payload);
+        if (state.originLinkedBounds) {
+          const difference =
+            state.originTime && state.originUpperBoundTime
+              ? time2sec(state.originUpperBoundTime) - time2sec(state.originTime)
+              : ORIGIN_TIME_BOUND_DEFAULT_DIFFERENCE;
+          state.originTime = sec2time(newOriginUpperBoundTimeSeconds - difference);
+        }
+        if (state.originTime && time2sec(action.payload) < time2sec(state.originTime)) {
+          state.originUpperBoundTime = state.originTime;
+        } else {
+          state.originUpperBoundTime = action.payload;
+        }
+      }
+    },
+    toggleOriginLinkedBounds(state: Draft<S>) {
+      state.originLinkedBounds = !state.originLinkedBounds;
+    },
+    updateOriginDate(state: Draft<S>, action: PayloadAction<S['originDate']>) {
+      state.originDate = action.payload;
+    },
+    updateOriginUpperBoundDate(state: Draft<S>, action: PayloadAction<S['originUpperBoundDate']>) {
+      state.originUpperBoundDate = action.payload;
+    },
+    replaceVias(state: Draft<S>, action: PayloadAction<S['vias']>) {
+      state.vias = action.payload;
+    },
+    updateVias(state: Draft<S>, action: PayloadAction<PointOnMap>) {
```

Comment:
Can you left a comment here to rename this action  "addVia" ? 
```suggestion
    addVia(state: Draft<S>, action: PayloadAction<PointOnMap>) {
```